src/YoutubeDl.php

```diff
@@ -139,7 +139,10 @@ private function jsonDecode($data): array
 
     private function processDownload(Process $process): Video
     {
-        $videoData = $this->jsonDecode(trim($process->getOutput()));
+        $jsonOutput = str_replace(['"','%(ext)s'],['','info.json'],$this->options['output']);
```

Comment:
This should handle all the cases explained in https://github.com/rg3/youtube-dl#output-template